generators/guestcontainer/templates/deploy/install-sfct.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Would you mind updating the original install/uninstall files, instead of adding the new files?